cmd/secrets/backend_selector.go

```diff
@@ -0,0 +1,57 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// +build secrets
+
+package secrets
+
+import (
+	"fmt"
+	"github.com/DataDog/datadog-agent/pkg/util/kubernetes/apiserver"
+	"time"
+)
+
+type backendPrefix string
```

Comment:
nit: this might be an overkill in type enforcement

Reply:
Not sure about this. I agree in Go this is not as useful, but I think it's nice to define this as constants instead of using plain strings.

One benefit is that in a refactor you only need to change the value of the constant instead of all the strings like "file" or "k8s_secret" across the codebase.